upload_settings_dialog.py

```diff
@@ -88,44 +81,51 @@ def __init__(self, upload_size, iface, project_definition):
             self.ui.license_cbx.setCurrentIndex(
                 self.ui.license_cbx.findText(DEFAULT_LICENSE[0]))
 
-        self.ui.update_chk.setEnabled(
-            'platform_layer_id' in self.project_definition)
-        self.ui.update_chk.setChecked(
-            'platform_layer_id' in self.project_definition)
-        self.set_head_msg()
-
-    def set_head_msg(self):
+        self.platform_layer_id = False
         if 'platform_layer_id' in self.project_definition:
-            platform_layer_id = self.project_definition['platform_layer_id']
-            self.head_msg_update = (
+            self.platform_layer_id = self.project_definition[
+                'platform_layer_id']
+
+        self.exists_on_platform = bool(self.platform_layer_id)
+        self.do_update = False
+
+        self.ui.update_radio.setEnabled(self.exists_on_platform)
+        self.ui.update_radio.setChecked(self.exists_on_platform)
+        self.set_labels()
+
+    def set_labels(self):
+        self.ui.situation_lbl.setVisible(self.exists_on_platform)
+        self.ui.question_lbl.setVisible(self.exists_on_platform)
+        for button in self.ui.upload_action.buttons():
+            button.setVisible(self.exists_on_platform)
+
+        if self.ui.update_radio.isChecked():
+            explaination_lbl = tr(
                 'The current project definition will be added to the '
                 'OpenQuake Platform project\nidentified as "%s"'
-                % platform_layer_id)
+                % self.platform_layer_id)
+            title_lbl = tr('Project title')
+            description_lbl = tr('Project description')
         else:
-            self.head_msg_update = ''
-        self.head_msg_new = (
-            'The project will be uploaded to the OpenQuake Platform.'
-            '\n\n(About %s MB of data will be transmitted)'
-            % self.upload_size)
-        if self.ui.update_chk.isChecked():
-            self.ui.head_msg_lbl.setText(self.head_msg_update)
-        else:
-            self.ui.head_msg_lbl.setText(self.head_msg_new)
+            explaination_lbl = tr(
+                'A new layer will be created on the OpenQuake Platform.'
+                '\n(About %s MB of data will be transmitted)'
+                % self.upload_size)
+            title_lbl = tr('New layer title')
+            description_lbl = tr('New layer description')
```

Comment:
I would change this into `New layer abstract` (because it will be written into the 'Abstract' metadata field)